magenta-lib/src/main/scala/magenta/logging.scala

```diff
@@ -190,4 +187,8 @@ case object Message extends Enum[Message] with PlayJsonEnum[Message] {
   case class Warning(text: String) extends Message
 
   val values = findValues
+}
+
+trait Loggable {
```

Comment:
A basic `Loggable` trait as a drop-in replacement for the same-named trait in Guardian Management.